tools/build_defs/apple/swift.bzl

```diff
@@ -382,8 +424,11 @@ def _swift_library_impl(ctx):
   ctx.file_action(output=swiftc_output_map_file, content=swiftc_output_map.to_json())
 
   args = _swift_xcrun_args(ctx) + ["swiftc"] + swiftc_args(ctx)
+  args += sorted(set(["-I" + d.dirname for d in dep_modules + [output_module]]))
+
   args += [
-      "-I" + output_module.dirname,
+      "-Xcc", "-fmodule-map-file=" + unextended_modulemap.path,
+      "-import-underlying-module",
```

Comment:
This will error out if there's no module with the same name as this swift_library. In most of the cases there won't be one.

In addition, using this with a giant modulemap would mean that all Objective-C symbols in the deps of this module are exposed to the clients, right?